src/MCPClient/lib/clientScripts/create_mets_v2.py

```diff
@@ -872,39 +873,61 @@ def getAMDSec(
     return ret
 
 
-def getIncludedStructMap(job, baseDirectoryPath, state):
+def _fixup_path_input_by_user(job, path):
+    """Fix-up paths submitted by a user, e.g. in custom structmap examples so
+    that they don't have to anticipate the Archivematica normalization process.
+    """
+    return os.path.join(
+        "", *[sanitizeName(name.encode("utf8")) for name in path.split(os.path.sep)]
```

Comment:
The idea here is that these files will already have been sanitized, so we're just matching up to the current state on the filesystem/database, correct?

Reply:
Sorry i missed this @cole, yep! That's right. We'll have to keep this in mind when you complete your work on that microservice job.